pyprof/nvtx/nvmarker.py

```diff
@@ -565,8 +565,7 @@ def patch_model_configs():
 
 
 def init(**kwargs):
```

Comment:
- Can you add a comment describing the valid key-value pairs.

Reply:
fixed